### PR TITLE
Feat/map 181/add preloaders when app fetch data

### PR DESCRIPTION
**Pull request by mazursasha1990**
Added preloaders to the app
Loader

### CODE REVIEW

**Review by naberalgamma — COMMENTED** [review]
maybe loader need add to map, when locations don't download, but maybe we need discus it with team
image

File: frontend/src/redux/reducers/isUserAuthorizedReducer.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import {
 import { isUserAuthorizedStateType } from '../ts-types';
 
 const initialState: isUserAuthorizedStateType = {
-  loading: true,
+  loading: false,
```
Comment (Yojuks) [inline frontend/src/redux/reducers/isUserAuthorizedReducer.tsx:8]: in this place why you changed this field?

**Review by Yojuks — APPROVED** [review]

File: frontend/src/redux/reducers/isUserAuthorizedReducer.tsx (hunk 1 above)
Comment (mazursasha1990) [reply frontend/src/redux/reducers/isUserAuthorizedReducer.tsx:8]: In my opinion, it is right when the initial loading state is false. I didn't find any issues after these changes. And, finally, I need the false state for the correct work of my task.

**Comment by mazursasha1990** [comment]
>

**Review by vasylneshchadimov — APPROVED** [review]

File: frontend/src/components/CircularLoader/CircularLoader.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import CircularProgress from '@mui/material/CircularProgress';
+import Box from '@mui/material/Box';
+
+export default function CircularIndeterminate() {
```
Comment (obozhko-dev) [inline frontend/src/components/CircularLoader/CircularLoader.tsx:5]: Why you use different names of components here in export and import to another file ? Here name `CircularIndeterminate` and in import we have `CircularLoader`. If it possible , I would rather use similar names in export and import

**Review by obozhko-dev — CHANGES_REQUESTED** [review]
1. in comment section we need to use preloader? this is question, I don`t know really it need. I set in console slow internet connection but comment is loaded fast, but I have only one comment :)
2. Photos in locations are loaded very slow, but for photo we need use something else maybe.
3. You can add preloder on the map when location list are preparing? or for this we need to use something from  leaflet ? here they https://reukraine.shtab.net/ used some preloder

**Review by obozhko-dev — APPROVED** [review]

**Pull request merged**